### ISSUE TITLE
Query methods isn't correct

### ISSUE THREAD

**Issue by rept** [kenn/active_flag#1]
Hi,

I believe the query method isn't correct.

Suppose you have this:

```
class Article < ApplicationRecord
flag :publisheds, [:nl, :fr, :en, :de]

```
And want to query articles that are published in nl.

You would expect: 
`Article.where_publisheds(:nl)`

To yield the results, however, if you have an article which is published in both English and Dutch (nl) then it won't show up, because the query that is being generated is:

` AND ("publisheds" = 1)`

This should instead be (at least for postgres): 
`where (publisheds & 1 > 0);`

which will contain all articles that are published for nl.

**Comment by kenn** [comment]
`=` is used only when the value has a single bit, and `&` for other cases.

See https://github.com/kenn/active_flag/blob/master/lib/active_flag.rb#L47

Am I missing something?

**Comment by rept** [comment]
Hi @kenn thanks for the swift reply.

Suppose this case:

```
a = Article.new
a.publisheds = [:nl, :fr]
a.subject = 'test'
a.save
```

In the database this will have publisheds at value 3

When I now want all articles that are published in nl I would do:

`Article.where_publisheds(:nl)`

This article won't appear however, because the query generated is:

`=> "SELECT "articles".* FROM "articles" WHERE ("publisheds" = 1)"`

I made a pull request which works fine on my end, please check.

**Comment by kenn** [comment]
Ohh, you are right! Thank you! 👍

**Comment by kenn** [comment]
Released as v1.4.0
